pydm/widgets/line_edit.py

```diff
@@ -109,9 +110,16 @@ def send_value(self):
                 if self._display_format_type == DisplayFormat.String:
                     self.send_value_signal[str].emit(send_value)
                 else:
-                    arr_value = list(filter(None, send_value.replace("[", "").replace("]", "").split(" ")))
+                    arr_value = list(filter(None, shlex.split(send_value.replace("[", "").replace("]", ""))))
                     arr_value = np.array(arr_value, dtype=self.subtype)
                     self.send_value_signal[np.ndarray].emit(arr_value)
+            elif self.channeltype == bool:
+                try:
+                    val = bool(distutils.util.strtobool(send_value))
```

Comment:
This is only my opinion, but it feels "off" to bring in a simple yet obscure function like this.

I'd recommend vendoring (copying in) this function instead of relying on distutils for this. 
Also, keep in mind that `distutils` itself is officially deprecated as of Python 3.10.